docs/README.md

```diff
@@ -157,12 +157,12 @@ dotnet_diagnostic.MA0036.severity = suggestion # MA0036: Make class static
 dotnet_diagnostic.MA0037.severity = error      # MA0037: Remove empty statement
 dotnet_diagnostic.MA0038.severity = suggestion # MA0038: Make method static
 dotnet_diagnostic.MA0039.severity = error      # MA0039: Do not write your own certificate validation method
-dotnet_diagnostic.MA0040.severity = suggestion # MA0040: Flow the cancellation token
+dotnet_diagnostic.MA0040.severity = suggestion # MA0040: Pass along the cancellation token
```

Comment:
I think you should copy the wording of https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2016